impl/pom.xml

```diff
@@ -219,7 +219,7 @@
             <id>tck-audit</id>
             <activation>
                 <property>
-                    <name>!skipTckAudit</name>
+                    <name>generateSource</name>
```

Comment:
Since it was named audit, we could just keep it as `tckAudit` or something similar.

Reply:
Ok,  updated and added comment about why it is needed but not active by default